app/components/common/Navigation.vue

```diff
@@ -429,8 +429,11 @@ export default Vue.extend({
                 source#logo-img.powered-by(srcset="/images/pages/base/logo.webp" type="image/webp")
                 img#logo-img.powered-by(src="/images/pages/base/logo.png" alt="CodeCombat logo")
               img.tecmilenio-logo(src="/images/pages/payment/tecmilenio-logo-2.png" alt="Tecmilenio logo")
-            a.navbar-brand(v-else-if="showHackStackLogo" :href="homeLink")
-              img#logo-img(src="/images/pages/base/logo+hs.png" alt="CodeCombat and HackStack logo")
+            a.navbar-brand(v-else-if="showHackStackLogo")
+              a(:href="homeLink")
+                img#logo-img(src="/images/pages/base/logo-smaller.png" alt="CodeCombat logo")
```

Comment:
actually there is already a image with Coco logo so you don't need to add a new file.

You can use this logic to add that image from above:
```
picture
    source#logo-img.powered-by(srcset="/images/pages/base/logo.webp" type="image/webp")
    img#logo-img.powered-by(src="/images/pages/base/logo.png" alt="CodeCombat logo")
```